### PR TITLE
fix(Tooltip):should hide when tooltip title is none

### PR DESCRIPTION
**Pull request by linhf123**
### 📦 Modified package

- [x] @oceanbase/design
- [ ] @oceanbase/ui
- [ ] @oceanbase/icons
- [ ] @oceanbase/charts
- [ ] @oceanbase/util
- [ ] @oceanbase/codemod
- [ ] Other (about what?)

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution
二封 Tooltip 时完全接管了 open，antd 内部 noTitle 时不调用 openChange 导致显示异常，同步了下 antd 的 noTitle 逻辑处理了下 innerOpen  取值

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     should hide when tooltip title is none      |
| 🇨🇳 Chinese |    当 tooltip title 不存在时应该不显示 tooltip       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Tests is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: packages/design/src/tooltip/index.tsx [hunk 1]
```diff
@@ -62,15 +63,9 @@ const Tooltip = React.forwardRef<TooltipRef, TooltipProps>(
...
 
-    const newOpen = open ?? visible ?? innerOpen;
-
-    useEffect(() => {
-      if (!isNil(open)) {
-        setInnerOpen(open);
-      } else if (!isNil(visible)) {
-        setInnerOpen(visible);
-      }
-    }, [open, visible]);
```
Comment (dengfuping) [inline packages/design/src/tooltip/index.tsx:73]: 单测挂了，这段状态维护逻辑应该保留？

**Review by dengfuping — CHANGES_REQUESTED** [review]

File: packages/design/src/tooltip/index.tsx (hunk 1 above)
Comment (linhf123) [reply packages/design/src/tooltip/index.tsx:73]: test 目录下新加了一个 util 没有运行测试用例报错了，我加个规则绕过一下

image

Comment (linhf123) [reply packages/design/src/tooltip/index.tsx:73]: 按照优先级来看，并不需要上面那段逻辑来同步 innerOpen，所以我直接去掉了
image

Comment (dengfuping) [reply packages/design/src/tooltip/index.tsx:73]: 单测 util 可以统一放 `tests/util.ts` 目录

**Pull request merged**